### ISSUE TITLE
Unable to pass all tests on Ubuntu 20.04

### ISSUE THREAD

**Issue by jserv** [fuhsnn/slimcc#88]
I attempted to run `make test-all` on Ubuntu 20.04, and the following errors raised:
```
c -std=c99 -g -fno-common -Wall -pedantic -Wno-switch -o stage2/slimcc \
    stage2/strings.o stage2/tokenize.o stage2/hashmap.o stage2/type.o stage2/main.o stage2/parse.o stage2/codegen.o stage2/preprocess.o stage2/unicode.o 
/usr/bin/ld: stage2/tokenize.o: relocation R_X86_64_32 against `.data' can not be used when making a PIE object; recompile with -fPIE
/usr/bin/ld: stage2/hashmap.o: relocation R_X86_64_32 against `.data' can not be used when making a PIE object; recompile with -fPIE
/usr/bin/ld: stage2/type.o: relocation R_X86_64_32 against `.data' can not be used when making a PIE object; recompile with -fPIE
/usr/bin/ld: stage2/main.o: relocation R_X86_64_32 against `.text' can not be used when making a PIE object; recompile with -fPIE
/usr/bin/ld: stage2/parse.o: relocation R_X86_64_32 against `.data' can not be used when making a PIE object; recompile with -fPIE
/usr/bin/ld: stage2/codegen.o: relocation R_X86_64_32 against `.data' can not be used when making a PIE object; recompile with -fPIE
/usr/bin/ld: stage2/preprocess.o: relocation R_X86_64_32 against `.data' can not be used when making a PIE object; recompile with -fPIE
/usr/bin/ld: stage2/unicode.o: relocation R_X86_64_32 against `.data' can not be used when making a PIE object; recompile with -fPIE
collect2: error: ld returned 1 exit status
make: *** [GNUmakefile:36: stage2/slimcc] Error 1
```

It seems that the option `-no-pie` was not properly specified.

**Comment by fuhsnn** [comment]
Should be fixed with the latest commit, thanks for pointing out.

**Comment by jserv** [comment]
> Should be fixed with the latest commit, thanks for pointing out.

Confirmed fix in commit ca2b46f7d19d9872d4860c75ba8e1caaf29bc87e
